platform/android/detect.py

```diff
@@ -178,7 +178,7 @@ def mySpawn(sh, escape, cmd, args, env):
 
     env.Append(CPPFLAGS=["-isystem", sysroot + "/usr/include"])
     env.Append(CPPFLAGS=string.split('-fpic -ffunction-sections -funwind-tables -fstack-protector-strong -fvisibility=hidden -fno-strict-aliasing'))
-    env.Append(CPPFLAGS=string.split('-DANDROID -DNO_STATVFS -DGLES2_ENABLED'))
+    env.Append(CPPFLAGS=string.split('-DANDROID_ENABLED -DNO_STATVFS -DGLES2_ENABLED'))
```

Comment:
That flag is actually already defined on line 230, so you can just remove the `-DANDROID` I guess.